### DISCUSSION TITLE
后台保活问题

### DISCUSSION THREAD

**Discussion by hua-sky** [constanline/XQuickEnergy#355] [General/我有一言当讲否]
大家都有什么好方法保活，以前和155一起开的时候，收能量很痛快，现在单开了，感觉怎么能量收不动了，手工启动下支付宝才开始收。
大家支个招！

**Comment by WskDnpl** [comment]
把支付宝加入电池白名单，后台自启打开

**Comment by charure** [comment]
你就是把1.55（不是修复板)和这芝麻粒一起装上，然后用1.55来保活吗？

**Reply by hua-sky** [reply]
现在不那样了，也许是支付宝版本的原因，开2个也不行，以前是可以的。
现在更简单些：直接xposed edge pro定时运行支付宝，不是shell命令运行，就是直接应用里的程序，在不解锁不亮屏的情况下也行。
shell命令运行的话只有3个进程，总是不收能量，手工启动下支付宝，进程数变6，开始收能量。这个不知道是不是支付宝版本的原因。反正现在这样能用，也不操心了。
多账号切换还没测试不解锁不亮屏的情况下的点击是否有效，切换到切换账号页面是可以实现的。目前我是自动亮屏解锁操作的。

**Reply by charure** [reply]
谢谢回复。
我可能没办法用edge po。
不过我受到你的启发，用LSPATCH在无root环境，用1.55修复版+芝麻粒，好像保活成功了。
芝麻粒里选的用闹钟唤醒。
